tests/benchmark/time.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "fibonacci:0.3.0"
+
+LANGS="c cpp rust"
+
+for lang in $LANGS; do
+	echo "$lang"
+
+	for _ in {1..10}; do
+		/usr/bin/time --format "%e %M" enarx deploy "examples/fibonacci-${lang}:0.3.0" 2>&1 1>/dev/null
```

Comment:
```suggestion
		time --format "%e %M" enarx deploy "examples/fibonacci-${lang}:0.3.0" 2>&1 1>/dev/null
```

Reply:
i think /usr/bin/time and time run different commands?